docs/wiki/getting-started/hardware.mdx

```diff
@@ -0,0 +1,128 @@
+---
+sidebar_position: 2
+---
+
+# Hardware
+
+There's a lot of parts that go into making FCs function the way they do, with different sensors, power management and outputs. This page will go over the different parts of a FC, what they do, and which parts are being used.
+
+## Main Components & General IO
+
+### MCU
+
+MCU (Microcontroller Unit) is the main chip on the FC, you can think of it as the "brain". It's responsible for running the flight controller software, and processing all the data from the sensors. From those inputs it does a bunch of ~~magic~~ math to generating the signals that go to the ESCs and run the motors.
```

Comment:
`From those inputs, it computes the signals that go to the ESCs and run the motors.`
please take my opinion with a grain of salt, afterall they are only opinion.
i chose `compute` rather than `calculate` based on this resource. https://jakubmarian.com/difference-between-compute-and-calculate/